### PR TITLE
Update signing-in.rst

### PR DESCRIPTION
**Pull request by wiersgallak**
#### Summary

Updated signing to add the requirement of needing an invite to the workspace prior to signing in.

### CODE REVIEW

File: source/help/getting-started/signing-in.rst [hunk 1]
```diff
@@ -1,9 +1,14 @@
 Signing in
 ==========
 
---------------
+Accessing your Mattermost workspace
+------------------------------------
 
-To sign in, navigate to the Mattermost sign-in page. You can get the URL of the sign-in page from your administrator.
+An ivite is required to access Mattermost. A member of your Mattermost workspace can send an invitation through an email or by providing you a link.
```
Comment (justinegeffen) [inline source/help/getting-started/signing-in.rst:7]:
```suggestion
An invite is required to access Mattermost. A member of your Mattermost workspace can send an invitation through an email or by providing you a link.
```

File: source/help/getting-started/signing-in.rst [hunk 2]
```diff
@@ -1,9 +1,14 @@
 Signing in
 ==========
 
---------------
+Accessing your Mattermost workspace
```
Comment (justinegeffen) [inline source/help/getting-started/signing-in.rst:4]:
```suggestion
Accessing your Mattermost Workspace
```

File: source/help/getting-started/signing-in.rst [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
+------------------------------------
 
-To sign in, navigate to the Mattermost sign-in page. You can get the URL of the sign-in page from your administrator.
+An ivite is required to access Mattermost. A member of your Mattermost workspace can send an invitation through an email or by providing you a link. 
+
+Sign In 
+--------
+To sign in, navigate to the Mattermost sign-in page. You can get the URL of the sign-in page from your administrator or in an invitation email. We recommend bookmarking the site so you can easily find it again.
```
Comment (justinegeffen) [inline source/help/getting-started/signing-in.rst:11]:
```suggestion

To sign in, navigate to the Mattermost sign-in page. You can get the URL of the sign-in page from your System Admin or in an invitation email. We recommend bookmarking the site so you can easily find it again.
```

File: source/help/getting-started/signing-in.rst [hunk 4]
```diff
@@ -1,9 +1,14 @@
 Signing in
```
Comment (justinegeffen) [inline source/help/getting-started/signing-in.rst:1]:
```suggestion
Signing In
```

**Review by justinegeffen — COMMENTED** [review]
I'm reviewing this in Carrie's place as it's a holiday in Canada today. :)

File: source/help/getting-started/signing-in.rst (hunk 1 above)
Comment (cwarnermm) [reply source/help/getting-started/signing-in.rst]: I committed this suggestion in error.

File: source/help/getting-started/signing-in.rst [hunk 5]
```diff
@@ -1,9 +1,14 @@
...
+------------------------------------
 
-To sign in, navigate to the Mattermost sign-in page. You can get the URL of the sign-in page from your administrator.
+An ivite is required to access Mattermost. A member of your Mattermost workspace can send an invitation through an email or by providing you a link. 
+
+Sign In 
+--------
+To sign in, navigate to the Mattermost sign-in page. You can get the URL of the sign-in page from your administrator or in an invitation email. We recommend bookmarking the site so you can easily find it again. 
 
 After signing in, the team that appears first on your team sidebar will open.  If you have not joined a team, the Team Selection
```
Comment (cwarnermm) [inline source/help/getting-started/signing-in.rst:13]:
```suggestion
After signing in, the team that appears first on your team sidebar will open. If you have not joined a team, the Team Selection
```

**Review by cwarnermm — APPROVED** [review]
Proposed updates look great!

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Gentle reminder on this, @esethna. :)

File: source/help/getting-started/signing-in.rst [hunk 6]
```diff
@@ -97,14 +90,14 @@ Mattermost.
 Switching Teams
 ---------------
 
-You can switch between teams you have joined using the team sidebar
-that appears left of your channel list on the left-hand sidebar.
+You can switch between teams you have joined using the team sidebar that appears to the left of your channel list on the left-hand sidebar.
 
 .. image:: ../../images/team-sidebar.png
```
Comment (esethna) [inline source/help/getting-started/signing-in.rst:95]: @cwarnermm separate from this PR these images should be updated

File: source/help/getting-started/signing-in.rst [hunk 7]
```diff
@@ -97,14 +90,14 @@ Mattermost.
...
 
 Setting Your Status
 -------------------
-You can set your status as online, away, do not disturb, or offline by selecting a status from the menu that appears when you click your avatar at the top of the channel list. Do not disturb disables desktop, email and push notifications.
+
+You can set your status as **Online**, **Away**, **Do not disturb**, or **Offline** by choosing a status from the menu that appears when you select your avatar at the top of the channel list. **Do not disturb** disables desktop, email, and push notifications.
 
 .. image:: ../../images/avatar-online-status-218x247.png
```
Comment (esethna) [inline source/help/getting-started/signing-in.rst:102]: @cwarnermm separate from this PR these images should be updated

File: source/help/getting-started/signing-in.rst (hunk 6 above)
Comment (cwarnermm) [reply source/help/getting-started/signing-in.rst:95]: Thanks, @esethna. I've logged a docs ticket (https://mattermost.atlassian.net/browse/MM-33036) for this update.

**Review by esethna — APPROVED** [review]

**Pull request merged**